src/test/java/Pages/Hotels/LandingPage.java

```diff
@@ -39,20 +57,64 @@ public class LandingPage extends Commands {
     By destinationInputLocator = By.id("location-field-destination");
     By destinationSuggestions = By.xpath("//div[@class='uitk-typeahead-results']//div[contains(@class,'truncat') and not(contains(@class,'uitk'))]");
 
+    By sighInLocator = By.xpath("//button[contains(text(), 'Sign in' ) and @type='button']");
+    By signInLinkInMenuBar = By.linkText("Sign in");
+
+    By checkOutDaysLocator = By.xpath("//h2[text()='June 2022']/following-sibling::table//button[@data-day]");
+
+    public void selectCheckoutDay(String day) {
+        List<WebElement> daysToSelect = findWebElements(checkOutDaysLocator);
+        for (WebElement value : daysToSelect) {
+            if (value.getAttribute("data-day").equals(day)) {
+                value.click();
+            }
+        }
+    }
+
+    // Calendar day adding methods
+    public void clickNextMonthBtn() {
+        clickIt(nextMonthBtnLocator);
+        Misc.pause(1);
```

Comment:
Instead of every hard coded wait, you can use explicit wait like waitUntilElementVisible or waitUntilElementClickable and pass the locator of the element that you waiting for.